clang/include/clang/Interpreter/Interpreter.h

```diff
@@ -18,6 +18,7 @@
 #include "clang/AST/GlobalDecl.h"
 #include "clang/Interpreter/PartialTranslationUnit.h"
 #include "clang/Interpreter/Value.h"
+#include "clang/Sema/CodeCompleteConsumer.h"
```

Comment:
We do not seem to need the definition of `CodeCompleteConsumer` maybe move the include file in the implementation?